### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by tamalsaha** [TimWolla/haproxy-auth-request#47] [Show and tell]
Hi Tim,
We are [using](https://github.com/appscode/voyager/blob/master/docs/guides/ingress/security/oauth.md) this project as part of our Kubernetes ingress controller [Voyager](https://github.com/appscode/voyager) . Just wanted to say thank you for the great work you have done here.

Regards,
-Tamal

**Comment by TimWolla** [comment]
Tamal,

while I know nothing about Kubernetes it's nice to see that people other than me actually use this. Feel free to create issues (or even pull requests) if there is anything to make your life easier :smile: 

May I ask how you learned about this project? Hacker News? Haproxy List? Google search?

**Comment by tamalsaha** [comment]
One of my coworkers, @diptadas was researching how to do oauth with HAProxy. He came across this repo via Google search.
